insta/src/runtime.rs

```diff
@@ -228,11 +255,12 @@ struct SnapshotAssertionContext<'a> {
     assertion_file: &'a str,
     assertion_line: u32,
     is_doctest: bool,
+    snapshot_type: SnapshotType,
```

Comment:
nit: more idimoatic to name this `snapshot_kind`, and will become part of the public API